packages/i18n/locales/cs/translation.json

```diff
@@ -0,0 +1,629 @@
+      "only_passed": "Pouze schválené návrhy"
+    },
+    "emoji": {
+      "baby": "Baby",
+      "ballotBox": "Ballot box",
+      "box": "Box",
+      "chart": "Chart",
+      "clock": "Clock",
+      "family": "Family",
+      "gear": "Gear",
+      "handshake": "Handshake",
+      "herb": "Herb",
+      "image": "Image",
+      "info": "Info",
+      "megaphone": "Megaphone",
+      "money": "Money",
+      "mushroom": "Mushroom",
+      "people": "People",
+      "recycle": "Recycle",
+      "robot": "Robot",
+      "swords": "Swords",
+      "token": "Token",
+      "whale": "Whale",
+      "x": "X",
+      "yinYang": "Yin Yang"
+    },
+    "error": {
+      "addressNotAMember": "Tato adresa není členem DAO.",
+      "cannotStakeMoreThanYouHave": "Nemůžete zamknout/odemknout víc tokenů než kolik máte.",
+      "cannotTxZeroTokens": "Nemůžete zamknout/odemknout 0 tokenů.",
+      "cantSpendMoreThanTreasury": "Nemůžete utratit víc než kolik máte v poklaně ({{amount}} ${{tokenSymbol}}).",
+      "checkInternetOrTryAgain": "Zkontrolujte si internetové připojení nebo to zkuste později.",
+      "connectWalletToContinue": "Pro pokračování připojte peněženku.",
+      "connectWalletToCreate": "Pro vytvoření DAO připojte peněženku.",
+      "couldntFindDAO": "Nenašli jsme vámi hledané DAO <2>home page</2>.",
+      "couldntFindProposal": "Nenašli jsme vámi hledaný návrh <2>DAO's home page</2>.",
+      "createProposalSubmitInvalid": "Před náhledem nebo publikováním opravte výše uvedené chyby.",
+      "daoFeatureUnsupported": "{{name}} zatím nepodporuje {{feature}}.",
+      "daoIsPaused": "Nemůžete vytvořit návrh když je DAO stopnuté",
+      "descriptionTooLong": "Popis vašeho DAO nesmí překračovat {{max}} znaků.",
+      "errorOccurredOnPage": "Omlouváme se, na této stránce došlo k chybě.",
+      "failedParsingCoreVersion": "Nepodařilo se určit hlavní verzi.",
+      "failedToGetTokenInfo": "Nepodařilo se získat informace o tokenu. Zadejte prosím platnou CW20 adresu.",
+      "govTokenBalancesDoNotSumTo100": "Celkové procento distribuce tokenů se musí rovnat 100 %, ale v současné době činí {{totalPercent}}.",
+      "insufficientWalletBalance": "Nedostatečný zůstatek v peněžence (celkem: {{amount}} ${{tokenSymbol}}).",
+      "invalidCosmosMessage": "Neplatná <2>Cosmos message</2>",
+      "invalidDuplicateFound": "Neplatné. Kopie nalezena v jiném seznamu.",
+      "invalidExecuteMessage": "Neplatná zpráva o provedení",
+      "invalidMessage": "Neplatná zpráva",
+      "loadingData": "Nepodařilo se načíst data.",
+      "mustBeMemberToCreateProposal": "Pro vytvoření návrhu musíte být členem.",
+      "nameIncorrectLength": "Název vašeho DAO musí mít něco mezi {{min}} a {{max}} znaky.",
+      "noClaimsAvailable": "Není k dispozici žádný výběr.",
+      "noMembers": "Do vašeho DAO jste nepřidal žádné členy.",
+      "noNftsSelected": "Nebylo vybráno žádné NFT.",
+      "noVotingPower": "Nikomu jste nedal počet hlasů.",
```

Comment:
zní to divně. neznám originál. nemělo by tam být třeba "Nikomu jste nedal své hlasy"?

obecná poznámka. možná by bylo dobré tam dát "Nikomu jste nedal/a své hlasy" - to by se pak mělo upravit všude